tests/storage/storage.go

```diff
@@ -455,7 +455,7 @@ var _ = SIGDescribe("Storage", func() {
                                    mount -t virtiofs %s %s
                                    touch %s
                            `, virtiofsMountPath, fs.Name, virtiofsMountPath, virtiofsTestFile)
-				userData := fmt.Sprintf("%s\n%s", tests.GetFedoraToolsGuestAgentUserData(), mountVirtiofsCommands)
+				userData := mountVirtiofsCommands
```

Comment:
just use please mountVirtiofsCommands directly? (same for line 516)

Reply:
done